@shared/api/package.json

```diff
@@ -8,7 +8,7 @@
     "bignumber.js": "^9.1.1",
     "prettier": "^2.0.5",
     "stellar-sdk": "yarn:stellar-sdk@^12.1.0",
-    "stellar-sdk-next": "yarn:stellar-sdk@12.1.0",
+    "stellar-sdk-next": "yarn:stellar-sdk@^12.1.0",
```

Comment:
we were getting a type mismatch bc without the ^, we were getting different versions of the SDK here